src/main/java/com/easypost/model/BaseAddress.java

```diff
@@ -0,0 +1,215 @@
+package com.easypost.model;
+
+import com.easypost.net.EasyPostResource;
+
+public class BaseAddress extends EasyPostResource {
+    private String id;
+    private String name;
+    private String company;
+    private String street1;
+    private String street2;
+    private String zip;
+    private String city;
+    private String state;
+    private String country;
```

Comment:
We should list these in the order of a real address:

street1
street2
city
state
zip
country

This goes for the getters/setters too.

Reply:
fixed